patches/com/mojang/blaze3d/platform/DepthTestFunction.java.patch

```diff
@@ -0,0 +1,10 @@
+--- a/com/mojang/blaze3d/platform/DepthTestFunction.java
++++ b/com/mojang/blaze3d/platform/DepthTestFunction.java
+@@ -6,6 +_,7 @@
+ 
+ @OnlyIn(Dist.CLIENT)
+ @DontObfuscate
++@net.neoforged.neoforge.internal.NonExhaustiveEnum(reason = "Remaining depth test functions will be added")
```

Comment:
I would prefer if this comment was worded a bit more generic and passive, i.e. along the lines of "Further depth test functions may be added". The same applies to the other affected enums.